spec/services/proforma_service/export_task_spec.rb

```diff
@@ -165,75 +152,57 @@
       end
     end
 
-    context 'when exercise has multiple files with role reference implementation' do
+    xcontext 'when task has multiple files with role reference implementation' do
       let(:files) { build_list(:codeharbor_solution_file, 2) }
 
       it 'adds test node to tests node' do
         expect(xml.xpath('/task/model-solutions/model-solution')).to have(2).items
```

Comment:
Just asking: Why is this not the case anymore? You've put the context on hold, so this requires additional work?